virt_who/register.py

```diff
@@ -52,14 +52,6 @@ def rhsm_override_uuid(self, ssh):
         ret, output = self.runcmd(cmd, ssh)
         ret, output = self.runcmd('cat /etc/rhsm/facts/override_uuid.facts', ssh)
 
-    def rhsm_backup(self, ssh):
-        ret, output = self.runcmd("ls /backup/rhsm.conf", ssh)
-        if ret != 0 or "No such file or directory" in output:
-            cmd = "rm -rf /backup/; mkdir -p /backup/; cp /etc/rhsm/rhsm.conf /backup/"
-            self.runcmd(cmd, ssh)
-        else:
-            logger.info("rhsm.conf is backup already({0})".format(ssh['host']))
-
     def rhsm_recovery(self, ssh):
```

Comment:
the same as above

Reply:
Because there's an no-member error in the first check as the following image shows, it means that class `Base` not have a function called `rhsm_backup`. And class `Register` have implement the class `Base`, so I move this function to `Base` class. I think this can solve question and make no effect on the `Register` class using.

![图片](https://user-images.githubusercontent.com/50477615/225843371-1aeb3619-03e4-4bbb-97da-67d73e9630d6.png)

Reply:
Maybe adding the `import`  also can fix this issue, such as
```
            from virt_who import register
            register.Register.rhsm_backup()
```
What do you think of it?

Reply:
I think that may lead to another error. If use `register.Register.rhsm_backup()`, it need to instantiate a `register` first before using this function. But I think there don't need a `register` instant?